modules/tinymce/src/core/main/ts/mode/Disabled.ts

```diff
@@ -74,12 +74,54 @@ const allowedEvents: ReadonlyArray<string> = [ 'copy' ];
 
 const isAllowedEventInDisabledMode = (e: Event) => Arr.contains(allowedEvents, e.type);
 
-const getAnchorHrefOpt = (editor: Editor, elm: SugarElement<Node>): Optional<string> => {
-  const isRoot = (elm: SugarElement<Node>) => Compare.eq(elm, SugarElement.fromDom(editor.getBody()));
-  return SelectorFind.closest<HTMLAnchorElement>(elm, 'a', isRoot).bind((a) => Attribute.getOpt(a, 'href'));
+const getAnchorHrefOpt = (element: SugarElement<Node>, rootElement: SugarElement<Node>): Optional<string> => {
+  const isRoot = (elm: SugarElement<Node>) => Compare.eq(elm, rootElement);
+  return SelectorFind.closest<HTMLAnchorElement>(element, 'a', isRoot).bind((a) => Attribute.getOpt(a, 'href'));
 };
 
 const processDisabledEvents = (editor: Editor, e: Event): void => {
+  if (handleAnchorClick(e, editor)) {
+    return;
+  }
+  if (handleSummaryClick(e, editor)) {
+    return;
+  }
+  if (isAllowedEventInDisabledMode(e)) {
+    editor.dispatch(e.type, e);
+  }
+};
+
+const handleSummaryClick = (e: Event, editor: Editor): boolean => {
+  /*
+    If an event is a click event on a summary element, then we want to prevent default browser behavior.
+    Accordions shouldn't be toggable in disabled editor.
+  */
+  const element = SugarElement.fromDom(e.target as Node);
+  const body = SugarElement.fromDom(editor.getBody());
+  if (isClickEvent(e)) {
+    return getClosestSummary(element, body).fold(Fun.never, (closestSummary) => {
```

Comment:
Couldn't this be just `exists`? Might be a bit weird to do prevent default in exists but it would be less code.